lib/fx/adapters/postgres/functions.rb

```diff
@@ -8,18 +8,37 @@ class Postgres
       class Functions
         # The SQL query used by F(x) to retrieve the functions considered
         # dumpable into `db/schema.rb`.
-        FUNCTIONS_WITH_DEFINITIONS_QUERY = <<-EOS.freeze
+        PG_11_FUNCTIONS_WITH_DEFINITIONS_QUERY = <<-SQL.squish.freeze
           SELECT
-              pp.proname AS name,
```

Comment:
Since we are reformatting these SQL calls, we can use pg_format(1):

```
SELECT
  pp.proname,
  pg_get_functiondef(pp.oid)
FROM
  pg_proc pp
  JOIN pg_namespace pn ON pn.oid = pp.pronamespace
  LEFT JOIN pg_depend pd ON pd.objid = pp.oid
    AND pd.deptype = 'e'
WHERE
  pn.nspname = 'public'
  AND pd.objid IS NULL
  AND pp.prokind != 'a'
ORDER BY
  pp.oid
```

Reply:
Looks much better, thank you!